persistence/pre_persistence/fisherman.go

```diff
@@ -281,7 +282,11 @@ func (m *PrePersistenceContext) SetFishermanPauseHeight(address []byte, height i
 	if !exists {
 		return fmt.Errorf("does not exist in world state")
 	}
-	fish.Paused = true
+	if height == 0 {
```

Comment:
If you end up going with a const per Olshansky's comment, remember to also replace this